src/components/Codec.js

```diff
@@ -40,9 +41,176 @@ export const Codec = ({ data, codec, title }) => {
                 '<video />'
             )
         ),
-        DrmSupports({ data, drmType: DrmType.WIDEVINE }),
-        DrmSupports({ data, drmType: DrmType.PLAYREADY }),
+        DrmSupports({
+            data,
+            drmType: DrmType.WIDEVINE,
+            levels: ['L1', 'L2', 'L3'],
+        }),
+        DrmSupports({
+            data,
+            drmType: DrmType.PLAYREADY,
+            levels: ['SL150', 'SL2000', 'SL3000'],
+        }),
         DrmSupports({ data, drmType: DrmType.PLAYREADY_LEGACY }),
         DrmSupports({ data, drmType: DrmType.FAIRPLAY })
     );
 };
+
+/**
+ *
+ * @param {CodecProps} props
+ * @returns {Element}
+ */
+export const Codec = ({ data, codec, title }) => {
+    return div(
+        { className: s['codec-wrapper'] },
+        div(
+            {
+                className: cx(
+                    s['title-wrapper'],
+                    s.box,
+                    s[getValidClass(data.mse || data.video)]
+                ),
+            },
+            div({ className: s.title }, title),
+            div(codec)
+        ),
+        div(
+            { className: s.half },
+            div(
+                {
+                    className: cx(
+                        s['text-center'],
+                        s['float-left'],
```

Comment:
do we honestly need to use `float:` with this layout? Can some `inline-block` not achieve the same thing?

Reply:
I had thought I was good for using floats